Mlem/App/Views/Shared/Labels/FullyQualifiedLabelView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  FullyQualifiedLabelView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-05-19.
+//
+
+import Foundation
+import MlemMiddleware
+import SwiftUI
+
+enum FullyQualifiedLabelStyle {
+    case small(showAvatar: Bool)
+    case medium(showAvatar: Bool)
+    case large(showAvatar: Bool)
```

Comment:
If `showAvatar` is used an argument for all three enum cases, might it be better to make the cases argument-less and have `showAvatar` as a standalone parameter of `FullyQualifiedLabelView`?

```swift
FullyQualifiedLabelView(entity: post.community, labelStyle: .small, showAvatar: true)